app-map.js

```diff
@@ -41,6 +41,10 @@ steal("can/util", "can/map", "can/compute", function(can){
 			this.__readyPromises.push(promise);
 			return promise;
 		},
+		pageStatus: function(statusCode, message) {
+			this.attr('statusCode', statusCode);
+			this.attr('statusMessage', message);
```

Comment:
big nitpick but we should probably be using the `this.attr({ })` form to prevent separate batches.